docs/content/documentation/schema/overview.md

```diff
@@ -49,20 +49,14 @@ An OSCAL file is organized as follows:
 {{% /usa-grid-row %}}
 {{% /usa-grid-container %}}
 
-{{<callout>}}
-**IMPORTANT NOTE TO DEVELOPERS**
-
-Every time the content of an OSCAL file changes, the following must also change:
-- A new UUID value must be generated and assigned to the root element's `uuid`.
-- The `last-modified` field in metadata must be assigned with the date and time at the moment the file is saved with the modified content.
-
-These are two mechanisms by which tools can quickly "know" if a file has changed since it was last encountered.
-
-When converting between formats, such as XML to JSON, these values should remain the same. This enables tools to know the content within the two formats is equivalent.{{</callout>}}
+<p />
+{{<partial "note-to-developers-uuid.html" >}}
 
 ### Identifier Use
 
-Identifiers uniquely identify content in OSCAL.
+Identifiers uniquely identify content in OSCAL. They are a critical to defining relationships between pieces of OSCAL content.
```

Comment:
In the sentence "They are a critical to defining relationships between pieces of OSCAL content."  please remove "a" in front of 'critical', or insert the missing noun after 'critical'.